test/unit/smartdown_content/smartdown_scenarios_test.rb

```diff
@@ -56,4 +57,15 @@ def outcome_is_reached(flow, answers, outcome)
                    "In flow #{flow.name} with answers #{answers}, outcome #{state.current_node.name} was reached and not #{outcome}"
     end
   end
+
+  def markers_are_included(flow, answers, markers)
```

Comment:
I think `markers_are_included` is reasonable, but it'd be worth renaming `same_elements? (or merging into the method), and using more explicit var names, like `expected_markers` and `actual_markers`.

The behaviour is "does `actual_markers` contain all of `expected_marker`, right? Which the current implementation of `same_elements?` does, eg `(['b','a'] - ['a', 'b','c', 'd']) => []` and `(['b','a', 'e'] - ['a', 'b','c', 'd']) => ['e']` (where the left array is expected, the right actual).

The logic is good, but the naming could be a bit clearer.

How about something like:

```ruby
def markers_are_included(flow, answers, expected_markers)
  state  = flow.state(true, answers)
  actual_markers = state.current_node.markers.map(&:marker_name)
  assert_empty
    (expected_markers - actual_markers),
    "In flow #{flow.name} with answers #{answers} markers #{markers} were expected but #{node_markers} was received"
end
```

(not sure if there is a better built in `assert` we could use there?